source/features/quick-mention.tsx

```diff
@@ -13,7 +13,10 @@ import features from '../feature-manager.js';
 import {getUsername, isArchivedRepoAsync} from '../github-helpers/index.js';
 import observe from '../helpers/selector-observer.js';
 
-const fieldSelector = 'textarea#new_comment_field';
+const fieldSelector = [
+	'textarea#new_comment_field',
+	'#react-issue-comment-composer textarea',
+];
```

Comment:
```suggestion
] as const;
```

This will make the $() below detect they're textareas without further assertions.

Reply:
Got it.